test/benchmark/lmbench/tcp_loopback_bw/config.json

```diff
@@ -4,5 +4,5 @@
     "search_pattern": "0.004096 ",
```

Comment:
Yes, but my concern is: Will this invalidate our historical data?

Reply:
The historical data should be stored on our CI machine. I think it can still be valid but may need to some additional operations(e.g., change the key or the directory name). May need the help from @grief8 later after this PR is merged.

Reply:
I don't know how historical data are kept if the benchmark name is changed. I guess there is some manual intervention. @grief is already on leave and thus not available for consulting. Let's keep the benchmark directory name unchanged in this PR and update it in the future.

Reply:
OK. Then I keep the directory name unchanged now. We can update it in the future as I also agree `tcp_loopback_bw_4k` is a better name.

Reply:
@tatetian , you probably meant @grief8 , not me